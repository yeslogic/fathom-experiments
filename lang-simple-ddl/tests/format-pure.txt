format test1 :=
  pure (A := Int64) 3;

format test2 :=
  pure (3 : Int64);

format test3 :=
  let x : Int64 := 3;
  pure x;
