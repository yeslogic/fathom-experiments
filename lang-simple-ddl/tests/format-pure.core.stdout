format test2 := #pure #Int64 3;

format test1 := #pure #Int64 3;

format test3 := let x <- #pure #Int64 3;
                #pure #Int64 x;
